data/saas/config/shipstation_config.yml

```
@@ -0,0 +1,75 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Shipstation
+  type: shipstation
+  description: A sample schema representing the Shipstation connector for Fides
+  version: 0.1.1
+
+  connector_params:
+    - name: api_key
+    - name: api_secret
```

A value like `api_key` will be converted to `Api key` in the UI, but it should be `API key`. To do this, we usually provide the `label` value. Also, any secret or sensitive value needs to be flagged so it's not displayed in the UI or exposed via the API.

```suggestion
  connector_params:
    - name: api_key
       label: API key
    - name: api_secret
       label: API secret
       sensitive: True
```

---

We also like to include `domain` and provide a default value. This is so the user can point to a different URL if needed.
```
  - name: domain
    default_value: ssapi.shipstation.com
```